### PR TITLE
U/fe/noop flag

### PR DESCRIPTION
**Pull request by frossie**
Sorry for the noisy pull request. I want to fix some tickets, so in preparation I regularised the indenting, fixed up the docs a bit, and added a noop (-n) flag (sometimes known as --pretend but didn't want to cause confusion with the -P option) that just runs through what it is going to do without doing it.

### CODE REVIEW

File: scripts/newinstall.sh [hunk 1]
```diff
@@ -34,23 +38,27 @@ LSST_HOME="$PWD"
...
-		;;
-	b)
-		batch_flag=true
-		;;
-	h)
-		help_flag=true
-		;;
-	P)
-		PYTHON=$OPTARG
+		c)
```
Comment (mjuric) [inline scripts/newinstall.sh]: I can live with this indenting scheme for a 'case', but I'm not sure it adds to readability (at the expense of an extra indent level).

File: scripts/newinstall.sh [hunk 2]
```diff
@@ -137,14 +155,14 @@ if true; then
 		WITH_ANACONDA=1
 	else
 		if [[ $PYVEROK != 1 ]]; then
-			cat <<-EOF
+		cat <<-EOF
```
Comment (mjuric) [inline scripts/newinstall.sh]: I think you want to keep the indent here (or indent the text by one more tab).

File: scripts/newinstall.sh [hunk 3]
```diff
@@ -188,69 +206,69 @@ if true; then
...
-			# Clone from git repository
-			git clone "$EUPS_GITREPO"
-			cd eups
-			git checkout $EUPS_GITREV
-		fi
-
-		./configure --prefix="$LSST_HOME"/eups --with-eups="$LSST_HOME" --with-python="$PYTHON"
-		make install
-
+	mkdir _build && cd _build
```
Comment (mjuric) [inline scripts/newinstall.sh]: I think you want to indent here to make it very clear all of this gets executed in a subshell (the parentheses)

File: scripts/newinstall.sh [hunk 4]
```diff
@@ -188,69 +206,69 @@ if true; then
...
-##########  Download optional component (python, git, ...)
+##########	Download optional component (python, git, ...)
 
 if true; then
 	if [[ $WITH_GIT = 1 ]]; then
 		echo "Installing git ... "
-		eups distrib install --repository="$EUPS_PKGROOT" git
-		setup git
-		CMD_SETUP_GIT='setup git'
+	$cmd eups distrib install --repository="$EUPS_PKGROOT" git
```
Comment (mjuric) [inline scripts/newinstall.sh]: Why no indent? This way it looks like it's not a part of the if [[]]; fi.

**Comment by mjuric** [comment]
Some minor comments inline, otherwise -- ship it.

**Comment by frossie** [comment]
Okay did them all and also some whitspace cleanup that needed doing. Left the case indented. Seems there is no strong guidance as to best practice and the editors seem to like indenting it.

**Pull request merged**